File: GreatestNo.c
#include<stdio.h>

int main()
{
    int a,b,c;
    printf("\nEnter 3 numbers (a,b,c) : ");
    scanf("%d %d %d",&a,&b,&c);
    if(a>b)
    {
        if(a>c)
            printf("Greatest No : a : %d",a);
        else if(a==c)
            printf("Greatest No : a&c are equal : %d ",a);
        else
            printf("Greatest No : c : %d",c);
    }
    else if(b>a)
    {
        if(b>c)
            printf("Greatest No : b : %d",b);
        else if(b==c)
            printf("Greatest No : b&c are equal : %d ",b);
        else
            printf("Greatest No : c : %d",c);
    }
    else
    {
        if(b>c)
            printf("Greatest No : a&b are equal : %d",a);
        else if(b==c)
            printf("Greatest No : a&b&c are equal : %d ",b);
        else
            printf("Greatest No : c : %d",c);
    }
    return 0;
}

File: GreatestNo.java
import java.util.*;

class GreatestNo
{
    public static void main(String args[])
    {
        Scanner ob1=new Scanner(System.in);
        int a,b,c;
        System.out.println("\nEnter 3 numbers (a,b,c) : ");
        a=ob1.nextInt();
        b=ob1.nextInt();
        c=ob1.nextInt();
        if(a>b)
        {
            if(a>c)
                System.out.println("Greatest No : a : "+a);
            else if(a==c)
                System.out.println("Greatest No : a&c are equal : "+a);
            else
                System.out.println("Greatest No : c : "+c);
        }
        else if(b>a)
        {
            if(b>c)
                System.out.println("Greatest No : b : "+b);
            else if(b==c)
                System.out.println("Greatest No : b&c are equal : "+b);
            else
                System.out.println("Greatest No : c : "+c);
        }
        else
        {
            if(b>c)
                System.out.println("Greatest No : a&b are equal : "+a);
            else if(b==c)
                System.out.println("Greatest No : a&b&c are equal : "+b);
            else
                System.out.println("Greatest No : c : "+c);
        }
    }
}
